hub/util/transform.py

```diff
@@ -0,0 +1,176 @@
+from hub.util.exceptions import InvalidTransformOutput
+from typing import Any, Callable, Dict, List, Optional, Set
+from hub.util.keys import get_chunk_key, get_index_meta_key, get_tensor_meta_key
+from hub.core.storage.provider import StorageProvider
+from hub.core.meta.index_meta import IndexMeta
+from hub.core.meta.tensor_meta import TensorMeta
+from hub.constants import CHUNK_MAX_SIZE, CHUNK_MIN_TARGET
+
+
+def transform_sample(sample: Any, fn_list: List[Callable], arg_list):
```

Comment:
```suggestion
def transform_sample(sample: Any, functions: Sequence[Callable], arg_list):
```

please use `Sequence` in places where appropriate instead of `List`. this is so the user can pass any iterable with a `__len__` method